### PR TITLE
High Voltage Lead

### PR DESCRIPTION
**Pull request by utilbird**
## About The Pull Request

shitsec moment!

## Why It's Good For The Game

realism, apparently 

## Changelog
:cl:
tweak: HoS egun has a chance of firing bullets instead of tazers
/:cl:

### CODE REVIEW

**Comment by VictorVow** [comment]
...why?

File: code/modules/projectiles/guns/energy/energy_gun.dm [hunk 1]
```diff
@@ -68,6 +68,19 @@
 	ammo_type = list(/obj/item/ammo_casing/energy/electrode/hos, /obj/item/ammo_casing/energy/laser/hos, /obj/item/ammo_casing/energy/disabler/hos)
 	ammo_x_offset = 4
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF
+	var/slippery = /obj/item/ammo_casing/energy/electrode/hos
+	var/butterfinger = /obj/item/ammo_casing/c9mm
+
+/obj/item/gun/energy/e_gun/hos/select_fire(mob/living/user)
+	if(ammo_type[switch] == butterfinger)
```
Comment (Archanial) [inline code/modules/projectiles/guns/energy/energy_gun.dm]: switch?

Comment (utilbird) [reply code/modules/projectiles/guns/energy/energy_gun.dm]: oops, forgot to push

**Comment by xDyouGamer** [comment]
Great PR,but you should make it be in affect only when female for more IRL accuracy https://www.youtube.com/watch?v=DVA95MfU9oo&ab_channel=ABC7

**Comment by OceanFish1** [comment]
Holy shit we need more PRs based on recent IRL situations

**Comment by ItsTisme** [comment]
> Holy shit we need more PRs based on recent IRL situations

This will age well

**Comment by DatBoiTim** [comment]
> > Holy shit we need more PRs based on recent IRL situations
> 
> This will age well

Now to add covid as a constant roundstart disease so everyone has to quarantine

**Review by ike709 — CHANGES_REQUESTED** [review]
Morbidly funny, but bad code and too high a probability.

**Comment by Sinestia** [comment]
Elon = Victor alt  i knew it

**Pull request closed without merging**